lib/parse-github-url.js

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
Much love to the Ebi project for this code

Reply:
Originally from [manage-github-apps](https://github.com/Financial-Times/ebi/pull/80) :)